src/amo/components/SearchForm/style.scss

```diff
@@ -0,0 +1,54 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+$item-padding: 5px 52px;
+
+.SearchForm-form {
+  background: $header-search-color;
+  border: 0;
+}
+
+.SearchInput-input,
+.SearchForm-form {
+  color: $base-color;
+  font-size: $font-size-m-smaller;
+  white-space: nowrap;
+}
+
+.SearchForm-suggestions {
+  &-list {
+    background-color: $white;
+    color: $black;
+    margin: 0;
+    padding: 0;
+  }
+
+  &-item {
+    cursor: pointer;
+    outline: none;
+    padding: $item-padding;
+    text-align: left;
```

Comment:
This should use the `@include text-align-start()` mixin so we get free RTL support.